package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Why do we need package.json and package-lock.json at all? My IDE uses the global eslint already.